Arduino/CN470RP1/TransmissionTestABPLoRa/TransmissionTestABPLoRa.ino

```diff
@@ -0,0 +1,111 @@
+
+#include <LoRaWan.h>
+// THOSE ARDUINO SAMPLE REQUIRES CHANGES IN THE ARDUINO CODE TO RUN DESCRIBED IN THE MAIN REPO
```

Comment:
@p-schuler @MaggieSalak please review this file as I added it

Reply:
Just reviewed. Looks good, I commited small blank line and identation fixes; I also added a comment about `setChannelOFF()` which doesn't compile for me in the Arduino IDE.

Reply:
Yes, you need a change in the arduino library, tracked in #1013